app/views/disbursements/_form.html.erb

```diff
@@ -3,20 +3,12 @@
 <%= form_with(model: disbursement, local: true, url: disbursements_path, html: { "x-data" => "{fee: null}" }) do |form| %>
   <%= form_errors(disbursement, "Disbursements") %>
 
-  <div class="field event-select-target">
-    <%= form.label :source_event_id, "From" %>
-    <%= form.collection_select(:source_event_id,
-                                @allowed_source_events,
-                                :id,
-                                current_user.admin? ? :admin_dropdown_description : :disbursement_dropdown_description,
-                                { prompt: "Select one…", disabled: ->(e) { e != @source_event && !policy(e).create_transfer? || (!current_user.admin? && e.balance_available <= 0) } },
-                                required: true,
-                                disabled: ) %>
-  </div>
-
   <div class="field event-select-target">
     <%= form.label :event_id, "To" %>
-    <%= form.select(:event_id,
+    <% if @source_event.plan.id == 4 %>
+      <%= form.text_field :event_id, placeholder: "hack_the_seas", required: true, disabled: %>
+    <% else %>
+      <%= form.select(:event_id,
```

Comment:
Imo we should keep the drop down and then have an "Other" option that then shows the text field